File: RepoTransBench/target_projects/Java/C#/doyensec_ajpfuzzer/public_tests/AJPTestCasesExtraPublicTest.cs
using System.Collections.Generic;
using Xunit;
using Doyensec.Ajpfuzzer;

namespace Doyensec.Ajpfuzzer.Tests.Public
{
    public class AJPTestCasesExtraPublicTest
    {
        [Fact]
        public void TestGetAllCasesNonEmpty()
        {
            IList<string> cases = AJPTestCases.GetAllCases();
            Assert.NotEmpty(cases);
        }

        [Fact]
        public void TestGetAllCasesContainsTest()
        {
            IList<string> cases = AJPTestCases.GetAllCases();
            bool found = false;
            foreach (var s in cases)
            {
                if (s.StartsWith("GET "))
                {
                    found = true;
                    break;
                }
            }
            Assert.True(found, "At least one case should start with 'GET '");
        }
    }
}

File: RepoTransBench/target_projects/Java/Go/doyensec_ajpfuzzer/public_tests/ajptestcases_extra_public_test.go
package public_tests

import (
	"ajpfuzzer"
	"strings"
	"testing"
)

func TestGetAllCasesNonEmpty(t *testing.T) {
	cases := ajpfuzzer.GetAllCases()
	if len(cases) == 0 {
		t.Error("Should not be empty")
	}
}

func TestGetAllCasesContainsTest(t *testing.T) {
	cases := ajpfuzzer.GetAllCases()
	found := false
	for _, s := range cases {
		if strings.HasPrefix(s, "GET ") {
			found = true
			break
		}
	}
	if !found {
		t.Error("At least one case should start with 'GET '")
	}
}